layouts/shortcodes/pipelines-compatibility.html

```diff
@@ -0,0 +1,7 @@
+<p><i>Due to 
+<a href="https://github.com/kubeflow/pipelines/issues/345">kubeflow/pipelines#345</a>
+and 
+<a href="https://github.com/kubeflow/pipelines/issues/337">kubeflow/pipelines#337</a>, 
+Kubeflow Pipelines depends on GCP services and some of the functionality is
```

Comment:
Done. I was in two minds about this one, because this inclusion is usually on a page which has already discussed GCP. But that's not guaranteed, and it does no harm to spell it out again.